mappers/map_dpa_article_to_dpa_design.js

```diff
@@ -0,0 +1,118 @@
+const cheerio = require('cheerio')
+const nanoid = require('nanoid').customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 10)
+
+module.exports = function (dpaArticle, images) {
+  const liArticle = []
+
+  // header
+  liArticle.push({
+    identifier: 'header',
+    id: `doc-${nanoid()}`,
+    content: {
+      dachzeile: dpaArticle.kicker,
+      headline: dpaArticle.headline,
+      subhead: dpaArticle.subhead,
+      byline: dpaArticle.byline,
+      dateline: dpaArticle.dateline,
+      creditline: dpaArticle.creditline
+    }
+  })
+  // teaser
+  if (dpaArticle.teaser) {
+    liArticle.push({
+      identifier: 'paragraph',
+      id: `doc-${nanoid()}`,
+      content: {
+        text: dpaArticle.teaser
+      }
+    })
+  }
+  // images
+  for (const image of images) {
+    liArticle.push({
+      identifier: 'image',
+      id: `doc-${nanoid()}`,
+      content: {
+        image: image.image,
+        caption: image.caption,
+        source: image.source
+      }
+    })
+  }
+  const $ = cheerio.load(dpaArticle.article_html)
```

Comment:
this is really difficult to read. Would be nice to have functions for each case.